.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+dist: trusty
+
+language: cpp
+
+compiler:  gcc
+
+before_install:
+- sudo apt-get update -qq
+- sudo apt-get install -qq libperl4-corelibs-perl libtool xorg-dev doxygen python-dev alien swig libgtest-dev libstdc++6 libxml2-dev gettext libxinerama-dev libxft-dev libxrandr-dev libxcursor-dev libgdbm-dev libc6 libc6-dev libmng-dev zlib1g-dev libcap-dev libpng12-0 libpng12-dev freeglut3-dev flex libx11-dev bison++ bisonc++ libjpeg-dev libjpeg8-dev python2.7 python2.7-dev python2.7-psycopg2 libogdi3.2-dev libgif-dev libxerces-c-dev libgeos-dev libgeos++-dev libfreetype6 libfreetype6-dev python-imaging libproj-dev python-setuptools libgif-dev libxerces-c-dev libcap-dev libpq-dev openssl libxml2-utils libxmu-dev
+
+
+before_script:
+  - cd earth_enterprise
+
+script:
+  - scons -j3 release=1 build >build.log
+  - cd src/NATIVE-REL-x86_64/bin/tests
+  - ./RunAllTests.pl
+
+cache:
+  directories:
+    - $HOME/earthenterprise/earth_enterprise/src/NATIVE-REL-x86_64/third_party
+
+notifications:
+  slack: opengee:YjzzzTnc7pgITPsrQPATIGwc
+  email:
+    recipients:
+      - elamar@t-sciences.com
```

Comment:
The notification mechanism is hard-coded;  multiple address can be specified, but there doesn't seem a way to discover who submitted a job and send an e-mail to that person.

Reply:
I just found this https://docs.travis-ci.com/user/notifications/ about notifications. See section 'How is the build email receiver determined? '. It seems that by default it send an email to the committer (if it can find their email) Or one has to hard code the emails (as you mentioned). I would want to have both options so an 'admin' would get notification but also the committer. This would avoid the 'admin' to pocking team members about failed builds.

Reply:
@tst-eclamar is there any way to get the email to work without putting in your personal email address?